### DISCUSSION TITLE
basset KO

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#1239] [Q&A (Help)] [answered]
From today cmd php artisan basset:cache go in timeout !!
What happen ?  Server issue ?

[a comment by an automated account is not shown]

**Comment by blondie63** [comment]
After all clear cmds php artisan basset:cache terminate in 62 seconds !!
Very long time, i think that CDNs have some problems.. or not ?
@tabacitu What you think ?

**Reply by karandatwani92** [reply, marked as answer]
Hey @blondie63 

Yes, the `unpkg` CDN is causing the issue. For now you can try:
```
php artisan basset:clear && php -d max_execution_time=120 artisan basset:cache
```

**Reply by blondie63** [reply]
> Hey @blondie63
> 
> Yes, the `unpkg` CDN is causing the issue. For now you can try:
> 
> ```
> php artisan basset:clear && php -d max_execution_time=120 artisan basset:cache
> ```

ok with 120 sec seems to be working but have you plan to change unpkg CDN ?

**Reply by pxpm** [reply]
We have plans of allowing you (the developer) to change the cdns, without having to overwrite the vendor files. 👍 And maybe, we can try replacing unpkg as it's clearly the most unreliable cdn we use.
